cloud

```diff
@@ -710,6 +720,9 @@ function scheduler_deploy() {
     # Deploy the jobs
     scheduler_job_pubsub_deploy "$project" "${prefix}trigger" \
                                 "$trigger_topic" '* * * * *' '{}'
+
+#    scheduler_job_pick_notification "$project" "${prefix}pick_notifications" \
+#                                     "$trigger_topic" '* * * * *' '{}'
```

Comment:
You don't need to copy the whole `scheduler_job_pubsub_deploy` function. You can just call `scheduler_job_pick_notification` function with the name of the topic you would create for the Cloud Scheduler to put its messages into (instead of `$trigger_topic`).